packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -1350,7 +1351,7 @@ export class LightdashAnalytics extends Analytics {
                 event: `${payload.event}.error`,
                 properties: {
                     ...payload.properties,
-                    error: e.message,
+                    error: getErrorMessage(e),
```

Comment:
this is the most common fix, wrap the error message in this function, that is handling unknown